app/code/Magento/Inventory/Model/SourceTypeLink.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model;
+
+use Magento\Framework\Model\AbstractExtensibleModel;
+use Magento\Inventory\Model\ResourceModel\SourceTypeLink as SourceTypeLinkResourceModel;
+use Magento\InventoryApi\Api\Data\SourceTypeLinkExtensionInterface;
+use Magento\InventoryApi\Api\Data\SourceTypeLinkInterface;
+
+/**
+ * {@inheritdoc}
+ *
+ * @codeCoverageIgnore
+ */
+class SourceTypeLink extends AbstractExtensibleModel implements SourceTypeLinkInterface
+{
+    /**
+     * @inheritdoc
+     */
+    protected function _construct()
```

Comment:
This doesn't seem to be a part of contract. Is there any specific usage for this method?